packages/command/messages/command.json

```diff
@@ -1,6 +1,6 @@
 {
-  "RequiresUsernameError": "This command requires a scratch org username set either with a flag or by default in the config.",
-  "RequiresDevhubUsernameError": "This command requires a dev hub org username set either with a flag or by default in the config.",
+  "RequiresUsernameError": "This command requires a username. Specify it with the -u parameter or by \"sfdx config:set defaultusername=<username>\".",
+  "RequiresDevhubUsernameError": "This command requires a dev hub username. Specify it with the -v parameter or by \"sfdx config:set defaultdevhubusername=<username>\".",
```

Comment:
ditto: change to "...or **with the** "sfdx config:set defaultdevhubusername=\<username\>" **command**."